system-tests/lib/cre/capabilities/evm/evm.go

```diff
@@ -28,6 +30,8 @@ import (
 
 const flag = cre.EVMCapability
 const configTemplate = `'{"chainId":{{.ChainID}},"network":"{{.NetworkFamily}}","logTriggerPollInterval":{{.LogTriggerPollInterval}}, "creForwarderAddress":"{{.CreForwarderAddress}}","receiverGasMinimum":{{.ReceiverGasMinimum}},"nodeAddress":"{{.NodeAddress}}"}'`
+const registrationRefresh = 20 * time.Second
+const registrationExpiry = 60 * time.Second
```

Comment:
both `registrationRefresh` and `registrationExpiry` are hardcoded values as @bolekk pointed out these values won't change per chain, and are the same in production.

If needed, we could externalize them in the TOML file